examples/load-local-bpmn-diagrams/index.html

```diff
@@ -0,0 +1,106 @@
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>BPMN Visualization - Load local BPMN diagrams</title>
+    <link rel="shortcut icon" href="../favicon.ico">
+    <link rel="stylesheet" href="../static/css/spectre.min.css" type="text/css">
+    <link rel="stylesheet" href="../static/css/icons.min.css" type="text/css">
+    <link rel="stylesheet" href="../static/css/main.css" type="text/css">
+    <style>
+        .relative {
+            position: relative;
+        }
+        .bpmn-viewport-container-absolute {
+            position: absolute;
+            top: 0;
+            left: 0;
+            right: 0;
+            bottom: 0;
+        }
+        #bpmn-viewport {
+            overflow-x: auto;
+            overflow-y: hidden;
+            padding-bottom: 100px;
+        }
+        .hidden {
+            display: none !important;
+        }
+    </style>
+</head>
+<body>
+
+<header class="navbar bg-dark p-2">
+    <section class="navbar-section ml-2">
+        <img src="../static/img/logo_64x64.ico" alt="logo" class="logo">
+        <span class="h3 text-light ml-2 lib-name-text">BPMN Visualization Examples</span>
+    </section>
+    <section class="navbar-section">
+        <a href="https://process-analytics.github.io/bpmn-visualization-js/" class="mr-2 btn btn-secondary">Docs</a>
+        <div class="dropdown mr-2">
+            <a href="#" class="btn btn-primary dropdown-toggle" tabindex="0">
+                GitHub <span class="icon icon-caret icon-in-the-middle"></span>
+            </a>
+            <ul class="menu">
+                <li><a href="https://github.com/process-analytics/bpmn-visualization-js" class="mr-2 btn btn-link">Library</a></li>
+                <li><a href="https://github.com/process-analytics/bpmn-visualization-examples" class="mr-2 btn btn-link">Examples</a></li>
+            </ul>
+        </div>
+    </section>
+</header>
+<section class="container col-10 flex-centered mt-2">
+        <section class="col-12 mt-2 relative">
+            <h2>Load local BPMN diagrams</h2>
+            <div id="file-selector">
+                <input type="file" id="btn-open-file" name="btn-open-file" class="hidden"/>
+                <label for="btn-open-file" class="btn btn-primary has-icon-right"><span>Choose a local file </span><span class="icon icon-upload mb-1"></span></label>
+            </div>
+<!--            TODO position at the right of the button-->
```

Comment:
@aibcmars can you help on this please? If we implement this change, the preview image must be updated

Reply:
done together, thanks